cliquet/__init__.py

```diff
@@ -16,6 +16,7 @@
 from cliquet.initialization import (  # NOQA
     initialize, initialize_cliquet, install_middlewares)
 
+__all__ = ['utils', 'errors', 'resource']
```

Comment:
The exhaustive list of everything we want to expose is the following:

- ProtectedResource
- BaseResource (Renamed Resource)
- register
- ProtectedViewSet
- ViewSet